langchain4j-core/src/main/java/dev/langchain4j/audit/api/listener/AiServiceInvocationErrorEventListener.java

```diff
@@ -0,0 +1,22 @@
+package dev.langchain4j.audit.api.listener;
+
+import dev.langchain4j.audit.api.event.AiServiceInvocationErrorEvent;
+
+/**
+ * A listener for {@link AiServiceInvocationErrorEvent}, which represents an event
+ * that occurs when an AI Service invocation fails.
+ * This interface extends the generic {@link AiServiceInvocationEventListener},
+ * specializing it for handling failure events.
+ * <p>
+ * Classes implementing this interface should handle scenarios where an AI Service
+ * invocation encounters an error. These scenarios include capturing and
+ * processing the associated error details encapsulated within the event.
+ */
+@FunctionalInterface
+public interface AiServiceInvocationErrorEventListener
```

Comment:
```suggestion
public interface AiServiceErrorListener
```
Seems to be clear, but much shorter, WDYT?

Reply:
Done